mmv1/products/networkconnectivity/PolicyBasedRoute.yaml

```diff
@@ -0,0 +1,194 @@
+      - !ruby/object:Api::Type::String
+        name: 'destRange'
+        default_value: "0.0.0.0/0"
+        description: |
+          The destination IP range of outgoing packets that this policy-based route applies to. Default is "0.0.0.0/0" if protocol version is IPv4.
+  - !ruby/object:Api::Type::Enum
+    name: 'nextHopOtherRoutes'
+    exactly_one_of:  # next_hop union
+      - next_hop_ilb_ip
+      - next_hop_other_routes
+    description: |
+      Other routes that will be referenced to determine the next hop of the packet.
+    values:
+      - :DEFAULT_ROUTING
+  - !ruby/object:Api::Type::String
+    name: 'nextHopIlbIp'
+    exactly_one_of:  # next_hop union
+      - next_hop_ilb_ip
+      - next_hop_other_routes
+    description: |
+      The IP address of a global-access-enabled L4 ILB that is the next hop for matching packets.
+  - !ruby/object:Api::Type::Integer
+    name: 'priority'
+    default_value: 1000
+    description: |
+      The priority of this policy-based route. Priority is used to break ties in cases where there are more than one matching policy-based routes found. In cases where multiple policy-based routes are matched, the one with the lowest-numbered priority value wins. The default value is 1000. The priority value must be from 1 to 65535, inclusive.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'virtualMachine'
+    exactly_one_of:  # target union
+      - virtual_machine
+      - interconnect_attachment
+    description: |
+      VM instances to which this policy-based route applies to.
+    properties:
+      - !ruby/object:Api::Type::Array
+        name: 'tags'
+        required: true  # optional in API docs, but that doesn't make sense here
+        description: |
+          A list of VM instance tags that this policy-based route applies to. VM instances that have ANY of tags specified here will install this PBR.
+        item_type: Api::Type::String
+  - !ruby/object:Api::Type::NestedObject
+    name: 'interconnectAttachment'
+    exactly_one_of:  # target union
+      - virtual_machine
+      - interconnect_attachment
+    description: |
+      The interconnect attachments that this policy-based route applies to.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'region'
+        required: true  # optional in API docs, but that doesn't make sense here
+        description: |
+          Cloud region to install this policy-based route on for Interconnect attachments. Use `all` to install it on all Interconnect attachments.
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    output: true
+    description: |
+      Time when the policy-based route was created.
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    output: true
+    description: |
+      Time when the policy-based route was created.
+  - !ruby/object:Api::Type::Time
+    name: 'kind'
+    output: true
+    description: |
+      Type of this resource.
```

Comment:
Do we need these fields in Terraform? especially `updateTime` and `kind`. Non-blockers though.

Reply:
Nope, we don't need them. I just erred on the side of including them.

Reply:
Makes sense. Including them is fine! I just want to confirm personally as I sometimes ask contributors to exclude these and want to make sure I didn't ask them to remove something that end up needed